header-footer-grid/templates/components/component-palette-switch.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Template used for component rendering wrapper.
+ *
+ * Name:    Header Footer Grid
+ *
+ * @version 1.0.0
+ * @package HFG
+ */
+namespace HFG;
+
+use HFG\Core\Components\PaletteSwitch;
+
+$icon_type = component_setting( PaletteSwitch::TOGGLE_ICON_ID );
+$svg_icon  = PaletteSwitch::get_icon( $icon_type );
+$label     = component_setting( PaletteSwitch::PLACEHOLDER_ID );
+?>
+<div class="toggle-palette">
+	<a href="<?php echo '#'; ?>" class="toggle palette-icon-wrapper">
```

Comment:
the HTML needs to be Accessible ready. please validate it against some accessibility testers

Reply:
I've ran this through `Lighthouse Accessibility` and it does not report any issues. Is there another tool that I should be using?